examples/src/main/java/ai/djl/examples/training/transferlearning/TransferFreshFruit.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.examples.training.transferlearning;
+
+import ai.djl.Model;
+import ai.djl.ModelException;
+import ai.djl.basicdataset.cv.classification.ImageFolder;
+import ai.djl.engine.Engine;
+import ai.djl.examples.training.util.Arguments;
+import ai.djl.metric.Metrics;
+import ai.djl.modality.cv.transform.OneHot;
+import ai.djl.modality.cv.transform.Resize;
+import ai.djl.modality.cv.transform.ToTensor;
+import ai.djl.modality.cv.transform.Transpose;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.types.Shape;
+import ai.djl.nn.Block;
+import ai.djl.nn.Parameter;
+import ai.djl.nn.SequentialBlock;
+import ai.djl.nn.core.Linear;
+import ai.djl.repository.Repository;
+import ai.djl.repository.zoo.Criteria;
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.training.DefaultTrainingConfig;
+import ai.djl.training.EasyTrain;
+import ai.djl.training.Trainer;
+import ai.djl.training.TrainingResult;
+import ai.djl.training.evaluator.Accuracy;
+import ai.djl.training.listener.SaveModelTrainingListener;
+import ai.djl.training.listener.TrainingListener;
+import ai.djl.training.loss.Loss;
+import ai.djl.training.optimizer.Adam;
+import ai.djl.training.optimizer.Optimizer;
+import ai.djl.training.tracker.FixedPerVarTracker;
+import ai.djl.training.util.ProgressBar;
+import ai.djl.translate.TranslateException;
+import ai.djl.util.Pair;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+
+public final class TransferFreshFruit {
+
+    private TransferFreshFruit() {}
+
+    public static void main(String[] args)
+            throws IOException, TranslateException, ModelException, URISyntaxException {
+        TransferFreshFruit.runExample(args);
+    }
+
+    public static TrainingResult runExample(String[] args)
+            throws IOException, TranslateException, ModelException, URISyntaxException {
+        Arguments arguments = new Arguments().parseArgs(args);
+        if (arguments == null) {
+            return null;
+        }
+
+        // Also available at
+        // "https://mlrepo.djl.ai/model/cv/image_classification/ai/djl/pytorch/resnet18_embedding/0.1/traced_resnet18_embedding.pt.gz";
+        String modelUrls = "/Users/fenkexin/Desktop/transferDJL/code/base_nw.pt";
```

Comment:
Why use local file path here?

Reply:
There is still some problem in the s3. It can be downloaded mannually but the code cannot find it yet.